packages/tizen_app_manager/tizen/src/tizen_app_manager_plugin.cc

```diff
@@ -76,6 +23,10 @@ void TizenAppManagerPlugin::RegisterWithRegistrar(
   registrar->AddPlugin(std::move(plugin));
 }
 
+flutter::EncodableList &TizenAppManagerPlugin::Applications() {
+  return this->applications_;
+}
+
 void TizenAppManagerPlugin::HandleMethodCall(
     const flutter::MethodCall<flutter::EncodableValue> &method_call,
     std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
```

Comment:
Prefer == over .compare() when comparing strings.
Please refer this.  https://github.com/flutter-tizen/plugins/issues/354
```cpp
const auto &method_name = method_call.method_name();
if (method_name == "wifiName") {
  ...
}
```

Reply:
The string comparison part of HandleMethodCall() has been modified to a string comparison method using ==.